views/package.json

```diff
@@ -9,6 +9,6 @@
     },
     "dependencies": {
         "@oat-sa/tao-item-runner": "^1.0.0",
-        "@oat-sa/tao-item-runner-qti": "^2.2.3"
+        "@oat-sa/tao-item-runner-qti": "^2.3.0"
```

Comment:
This is backport, so you'll need to release it *manually*, not with CI action.
https://oat-sa.atlassian.net/wiki/spaces/OAT/pages/114885777/How+to+backport
So you'll need to include in this PR:
- version bump for extension itself
- bundle css & js

Reply:
Thanks, done